Jenkinsfile

```diff
@@ -126,6 +125,111 @@ pipeline {
             // and resources
             failFast true
             parallel {
+                stage('Build RPM on CentOS 7') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile-mockbuild.centos:7'
```

Comment:
Filename not portable to all development platforms.

Reply:
Fixed.